src/common/conditional_compilation/tests/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ ov_add_test_target(
         DEPENDENCIES
         LINK_LIBRARIES
             gmock
-            funcTestUtils
+            commonTestUtils
```

Comment:
I don't see any dependency in code on these libraries. Can we remove this dependency?

Reply:
Yes, it depends only from `openvino::conditional_compilation`